### DISCUSSION TITLE
More thanks: Data Workspace Technical Documentation

### DISCUSSION THREAD

**Discussion by michalc** [ministryofjustice/mkdocs-tech-docs-template#24] [Show and tell]
Just another thank you - using this template we've made a start on the documentation for the DBT's data analysis platform, Data Workspace at https://data-workspace.docs.trade.gov.uk/ (standard caveat: still some rough edges)

**Comment by SoumayaMauthoorMOJ** [comment]
Love it - and happy to merge in any extra css changes you made if you think relevant!
